product_docs/docs/biganimal/release/using_cluster/pgd_defaults_ba.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: Postgres Distributed (PGD) presets on BigAnimal
+navTitle: PGD presets on BigAnimal
+deepToC: true
+---
+
+## Commit scope
+
+[Commit scopes](../../../pgd/latest/durability/commit-scopes/) in PGD is a set of rules that describes the behavior of the system as transactions are committed. Because they define how transactions are replicated across a distributed database, they impact consistency and reliability.
+
+The actual behavior depends on which kind of commit scope a commit scope's rule uses: [Group Commit](/../../../pgd/latest/durability/group-commit/), [Commit At Most Once](/../../../pgd/latest/durability/camo/), [Lag Control](/../../../pgd/latest/durability/lag-control/), [Synchronous Commit](/../../../pgd/latest/durability/synchronous_commit/), or a combination of these.
+
+Customizing these settings can be complex, presenting a challenge for users. 
```

Comment:
Not keen on this language. Suggests to the reader they need to be up to the task.

"This flexibility means that selecting a balanced combination of rules can take time. To speed up deployment, BigAnimal's PGD has a preset selection of commit_scopes for typical user requirements."

Reply:
subbed this in 👍